TOMsPlugin/manageRestrictionDetails.py

```diff
@@ -123,15 +126,21 @@ def __init__(self, tomsToolbar, proposalsManager):
         # Add actions to the toolbar
         self.tomsToolbar.addAction(self.actionSelectRestriction)
         self.tomsToolbar.addAction(self.actionRestrictionDetails)
-        self.tomsToolbar.addAction(self.actionCreateBayRestriction)
-        self.tomsToolbar.addAction(self.actionCreateLineRestriction)
-        self.tomsToolbar.addAction(self.actionCreatePolygonRestriction)
-        self.tomsToolbar.addAction(self.actionCreateSignRestriction)
-        self.tomsToolbar.addAction(self.actionRemoveRestriction)
-        self.tomsToolbar.addAction(self.actionEditRestriction)
-        self.tomsToolbar.addAction(self.actionSplitRestriction)
-        self.tomsToolbar.addAction(self.actionCreateConstructionLine)
-        self.tomsToolbar.addAction(self.actionImportRestrictions)
+
+        if self.user_elevation in [
+            "admin",
+            "write_confirm_operator",
+            "write_no_confirm_operator",
+        ]:
```

Comment:
Do you think it will be a better solution to have a UserElevation class (or a map) with the different permission per elevation? then we could call `self.user_elevation['can_print']` or something like that?

Reply:
What do you think of the system now?